CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.6.0 - 2018-03-08
+### Added
+- Configure deployment source to a GitHub repository (requires authorizing Azure to access GitHub)
+- Create PHP, .NET Core, and Ruby web apps on Linux
+- Faster zipdeploy by leveraging SCM_DO_BUILD_DURING_DEPLOYMENT app setting.  Learn more [here](https://aka.ms/Kwwkbd)
```

Comment:
It would be nice to put in an anchor at the correct spot in the readme file and link directly to it instead of to the entire README.

e.g. add <a name="faster-deployment"></a> to the README, then add #faster-deployment to the end of the forwarded link.

Reply:
I will change the aka.ms link to the anchor when this README is merged into master